pkg/skaffold/util/term.go

```diff
@@ -35,3 +44,25 @@ func IsTerminal(w io.Writer) (uintptr, bool) {
 
 	return 0, false
 }
+
+func SupportsColor() (bool, error) {
+	if runtime.GOOS == constants.Windows {
+		return true, nil
+	}
+
+	res, err := colors()
+	if err != nil {
+		return false, fmt.Errorf("checking terminal colors: %w", err)
+	}
+
+	numColors, err := strconv.Atoi(strings.TrimSpace(string(res)))
+	if err != nil {
+		return false, err
+	}
```

Comment:
This code is a bit hard to understand without looking up to see that `colors()` is an alias to `tputColors()`.  I think you can just inline `tputColors()` here.  And instead use `util.RunCmdOut()` to execute so that the test can use `testutil.FakeCmd`.

Reply:
done 👍🏼